src/lib/app/types/identity.ts

```diff
@@ -98,3 +99,8 @@ export type VerifiableCredentialTemplate = {
     credentialType: CredentialTypes
     userType: UserType
 }
+
+export type IdentityFilter = {
+    limitFilter: Filter
```

Comment:
this filter is not used please remove

Reply:
for sure it is used by applying the limit to the search but is no where added to switch between nor we want them currently to switch this. This could be added by having a dropdown where users can specify the limit between 30, 50 or 100 but we dont need this yet.

Same applies for the limitFilter of Streams

Reply:
Filter was removed